### PR TITLE
sriov pkg: Added create and newBuilder funcs for sriovoperatorconfig

### PR DESCRIPTION
**Pull request by evgenLevin**

### CODE REVIEW

File: pkg/sriov/operatorconfig.go [hunk 1]
```diff
@@ -29,6 +29,56 @@ type OperatorConfigBuilder struct {
 	errorMsg  string
 }
 
+// NewOperatorConfigBuilder creates new instance of OperatorConfigBuilder.
+func NewOperatorConfigBuilder(apiClient *clients.Settings, nsname string) *OperatorConfigBuilder {
+	glog.V(100).Infof(
+		"Initializing new OperatorConfigBuilder structure with the following params: %s", nsname)
```
Comment (achuzhoy) [inline pkg/sriov/operatorconfig.go:35]: Maybe it's worth to replace param with namespace.
Looking at the log - not clear that the param is nsname

Comment (cdvultur) [reply pkg/sriov/operatorconfig.go:35]: agree with @achuzhoy on this one.

File: pkg/sriov/operatorconfig.go [hunk 2]
```diff
@@ -29,6 +29,56 @@ type OperatorConfigBuilder struct {
...
+		)
+
+		if err != nil {
+			glog.V(100).Infof("Failed to create the SriovOperatorConfig")
+
+			return nil, err
+		}
+	}
+
+	builder.Object = builder.Definition
```
Comment (cdvultur) [inline pkg/sriov/operatorconfig.go:77]: Why do you overwrite the buidler.Object with builder.Definition ?

Comment (evgenLevin) [reply pkg/sriov/operatorconfig.go]: right, no need! removed

**Review by cdvultur — APPROVED** [review]
lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**